articles/support/tickets.md

```diff
@@ -1,20 +1,24 @@
 # Opening and Managing Tickets in Support Center
 
-With [Support Center](https://support.auth0.com), you can create various tickets for questions or issues you may be experiencing. You need to be a full dashboard administrator to access Support Center. 
+With [Support Center](https://support.auth0.com), you can create various tickets for questions or issues you may be experiencing. You need to be a full dashboard administrator to access Support Center.
 
 Existing Auth0 Appliance customers should create a new Auth0 cloud account to be able to login to Support Center (this account can also be used for dev/test purposes at no cost under your subscription). Request that account be associated with your existing subscription via your Customer Success Manager or Sales Executive.
 
 ## Opening a New Ticket
 
 1. From [Support Center](https://support.auth0.com), click on the **OPEN TICKET** button.
 ![](/media/articles/support/open-ticket.png)
+1. Choose the "Affected Account" from the dropdown menu.
```

Comment:
@solepano Yes, you are correct. :)

Markdown lists render in a number-agnostic way, so this displays as item number "2" when viewed.
I was following the convention in the rest of the file (it uses 1's throughout), but if that's wrong I can change it! :)